packages/api-contract/src/index.ts

```diff
@@ -8,4 +9,5 @@ const c = initContract();
 export const contract = c.router({
   post: postContract,
   user: userContract,
+  organizer: organizerContract,
```

Comment:
Should we export the attendee contract mai?
```suggestion
  organizer: organizerContract,
  attendee: attendeeContract
```